Sources/App/Core/Score.swift

```diff
@@ -85,11 +86,13 @@ enum Score {
             case 5..<20: score += 5
             default: score += 10
         }
+        
+        if candidate.hasTestTargets { score += 15 }
```

Comment:
```suggestion
        if candidate.hasTestTargets { score += 5 }
```

Before merging this, I did a little experimentation with the newly analysed target data on production.

We have tagged **4,632** (~75%) packages as having a test target in their current version. Of course, this is so high because, depending on how the package author created the package, it might have included a default test target at creation time.

For example: https://github.com/iballan/BasicServiceLocator/blob/master/Tests/BasicServiceLocatorTests/BasicServiceLocatorTests.swift

The good news is that it took me quite a while to find a package with this default template test intact and unedited. This is going to lift some packages with a test target and without tests, but it’s also going to lift all the packages with a test target _and_ tests.

As we have discussed, there is more in-depth analysis we could do to find other metadata that would give a better sense of whether tests exist. We will tackle that as a separate PR.

If this number were 99% or even 90%, I’d want to consider whether this PR was making a helpful contribution. At 75%, I want to merge it, but 15 points is too high a score adjustment for this level of analysis. I suggest we adjust it to 5 points.